blogposts/2022/sourcegraph-extension-for-vs-code.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'How I use the Sourcegraph extension for VS Code'
+description: 'Sourcegraph’s extension for Visual Studio Code allows you to search millions of open source repositories without cloning them to your local machine or leaving your IDE. Here’s how one engineer integrates it into his workflow.'
+author: Murat Sutunc
+publishDate: 2022-02-08T00:00-07:00
+tags: [blog]
+slug: ways-to-use-sourcegraph-extension-for-vs-code
+heroImage: https://sourcegraphstatic.com/blog/vs-code-extension/sourcegraph-vs-code-extension.png
+socialImage: https://sourcegraphstatic.com/blog/vs-code-extension/sourcegraph-vs-code-extension.png
+published: true
+---
+
+![Explore millions of open source repositories right from your IDE](https://sourcegraphstatic.com/blog/vs-code-extension/sourcegraph-vs-code-extension.png)
+
+My name is [Murat Sutunc](https://handbook.sourcegraph.com/team#murat-sutunc), and I’m the Engineering Manager for the Growth and Integrations team at Sourcegraph. Our team currently owns the initial user journey, growth-oriented projects, our code host and third-party integrations (including our IDE and browser extensions), and Sourcegraph extensions. As someone who does a lot of code reviews and occasionally jumps in to fix small issues, I wanted to share with everyone how I use my favorite IDE (VS Code!) and Sourcegraph together.
+
+TL;DR 👉 The Sourcegraph VS Code extension is a really powerful tool that lets you search your code and 2M+ open source repositories, all from your IDE! It’s available on [the marketplace](https://marketplace.visualstudio.com/items?itemName=sourcegraph.sourcegraph)—no account needed, give it a try!
+
+### Finding examples to fix issues
+
+Now and then, I jump in to help our team with some issues that are not on the critical path. For those unfamiliar, I think it's fair to say Sourcegraph's codebase is sizeable. It still fits on my laptop hard drive, but it’s large enough to tire IDEs when indexing. On top of our large main repository, we have several hundred additional repositories, and quickly finding what I'm looking for can be challenging.
```

Comment:
@muratsu Could we link to our codebase to demonstrate what you mean by sizeable? I'm sure people will be curious as you say it!

Reply:
```suggestion
Now and then, I jump in to help our team with some issues that are not on the critical path. For those unfamiliar, I think it's fair to say [Sourcegraph's codebase is sizeable](https://github.com/sourcegraph/sourcegraph). It still fits on my laptop hard drive, but it’s large enough to tire IDEs when indexing. On top of our large main repository, we have several hundred additional repositories, and quickly finding what I'm looking for can be challenging.
```

Reply:
Can we link to https://github.com/sourcegraph instead? I think the fact that we have 530 repositories is also important. Not only a med size primary codebase but also bits of code sits in multiple repos.